pyfolio/timeseries.py

```diff
@@ -741,38 +738,22 @@ def perf_stats(
 
     Returns
     -------
-    dict / pd.DataFrame
+    pd.Series
         Performance metrics.
 
     """
 
-    all_stats = OrderedDict()
-    all_stats['annual_return'] = annual_return(returns, period=period)
-    all_stats['annual_volatility'] = annual_volatility(returns, period=period)
-    all_stats['sharpe_ratio'] = sharpe_ratio(
-        returns)
-    all_stats['calmar_ratio'] = calmar_ratio(returns, period=period)
-    all_stats['stability'] = stability_of_timeseries(returns)
-    all_stats['max_drawdown'] = max_drawdown(returns)
-    all_stats['omega_ratio'] = omega_ratio(returns)
-    all_stats['sortino_ratio'] = sortino_ratio(returns)
-    all_stats['skewness'] = stats.skew(returns)
-    all_stats['kurtosis'] = stats.kurtosis(returns)
+    stats = pd.Series()
```

Comment:
This is considerably faster as:

```python
return pd.Series(OrderedDict(
    (stats_func.__name__, stats_func(returns)
    for stats_func in SIMPLE_STATS_FUNCS +
    (FACTOR_STAT_FUNCS if factor_returns else [])
))
```

Otherwise we resize the series everytime we add an element to the series.